init-service/pom.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">  
+  <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>uk.ac.ox.ndph.mts</groupId>
+        <artifactId>mts-services-parent</artifactId>
+        <version>0.0.1-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+  <groupId>uk.ac.ox.ndph.mts</groupId>
+  <artifactId>init-service</artifactId>
+  <version>0.0.1-SNAPSHOT</version>  
+  <name>init-service</name>
+  <description>Service for init orchestrator clinical trials</description>
```

Comment:
minor: this message seems like this service orchestrates the trials. where in fact its not orchestrating anything (imo) . a better description would be:
A process that prepare the different micro-services and populates them with initial configuration and data

Reply:
Done